lang/src/org/partiql/lang/util/CastToCastOts.kt

```diff
@@ -0,0 +1,185 @@
+package org.partiql.lang.util
+
+import com.amazon.ionelement.api.ionNull
+import org.partiql.lang.domains.PartiqlAst
+import org.partiql.lang.domains.PartiqlPhysical
+import org.partiql.pig.runtime.LongPrimitive
+
+/**
+ * Conventional type IDs
+ */
+class TypeId {
```

Comment:
The types defined will be gradually pulled out later as we pursue the pluggability goal of OTS.